Automattic/wp-calypso
# Redesigned DIFM New/Existing Site Step

#### Changes proposed in this Pull Request

* Redesigns the New/Existing Site screen according to the new designs in pdh1Xd-FG-p2
* I have deleted the custom components and replaced them with `IntentScreen` from the `@automattic/onboarding` package. 

<img width="1907" alt="image" src="https://user-images.githubusercontent.com/5436027/157604327-7d717785-31ba-4dc3-85ee-8d36d1b82f76.png">

**Loading animation**

https://user-images.githubusercontent.com/5436027/157638294-84055a9d-12b9-4765-b632-a58fdd91d0b2.mp4


#### Testing instructions



* Log in as a user who has more than 1 existing site.
* Go to `/start/do-it-for-me?flags=force-sympathy`. (`flags=force-sympathy` ensures that local browser storage is cleared)
* Confirm that you can see the placeholder while the product list is loading.
* Confirm that the screen matches the design shared in the link above.
* Confirm that the price for the DIFM product is displayed correctly according to the user currency.



Related to 0-as-1201912426423457/f
